frames/src/main/java/com/checkout/frames/component/paybutton/PayButtonViewModel.kt

```diff
@@ -66,9 +66,12 @@ internal class PayButtonViewModel @Inject constructor(
         // Get the BillingAddress value only if "the billing address is enabled" and "the address is edited".
         val address = billingAddress.value.takeIf { isBillingAddressEnabled.value && it.isEdited() }
 
+        // Get the cardholder name from payment form and if it is empty take it from the Billing address
+        val name = cardHolderName.value.ifEmpty { address?.name }
```

Comment:
is there any documentation we should mention that if the cardholder's name shows in BOTH payment form and billing address, it will prioritise the payment form over the billing address?

Reply:
Sure. We aim to document it in Readme later when we will plan to do the next release as it is in the development stage.

Reply:
of course! curious do we have anything to remind us of that this part will be documented in Readme?

Reply:
It's an AC on the ticket :)

Reply:
@jheng-hao-lin-cko Generally, when we do release, we decide what should we include in ReadMe and release notes from the tickets (ACS in the [PIMOB-2085 (ACS in the ticket description))](https://checkout.atlassian.net/browse/PIMOB-2085)  we have worked on. @patrick-hoban-cko can advise more on it. Additionally, Do you have any better idea to remind us, happy to follow it.